xml/System.Collections.Generic/IEnumerable`1.xml

```diff
@@ -64,7 +64,7 @@
     </remarks>
     <block subset="none" type="overrides">
       <para>
-        <see cref="T:System.Collections.Generic.IEnumerable`1" /> is included for parity with non-generic collections; implementing <see cref="T:System.Collections.Generic.IEnumerable`1" /> allows a generic collection to be passed to a method that expects an <see cref="T:System.Collections.IEnumerable" /> object.</para>
+        To remain compatible with methods that iterate non-generic collections, <see cref="T:System.Collections.Generic.IEnumerable`1" /> implements <see cref="T:System.Collections.IEnumerable" />. This allows a generic collection to be passed to a method that expects an <see cref="T:System.Collections.IEnumerable" /> object.</para>
```

Comment:
We could combine the para tag line with the text in one line if we're not starting with a tag.

```suggestion
      <para>To remain compatible with methods that iterate non-generic collections, <see cref="T:System.Collections.Generic.IEnumerable`1" /> implements <see cref="T:System.Collections.IEnumerable" />. This allows a generic collection to be passed to a method that expects an <see cref="T:System.Collections.IEnumerable" /> object.</para>
```